### PR TITLE
Rebaz36/issue52

### LINKED ISSUE #52: implement About Us Page
create the about us page using the proper components 
issue 4 about us page

### PR DESCRIPTION
**Pull request by rebaz36**
Fixes #52

### CODE REVIEW

File: components/Aboutus/WhyChooseUs.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default function WhyChooseUs({ title, subtitle, Logo }) {
+  return (
+    <div className="bg-white flex flex-row lg:p-4 md:p-2 rounded-xl place-content-center">
```
Comment (zahrayh) [inline components/Aboutus/WhyChooseUs.js:3]: Add `shadow-md` class to it and don't center align the elements in the card. Also reduce the width, they are too big

File: pages/aboutus.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  { title: "Posted", subtitle: "15k Vacancies", logo: FaPenNib },
+  { title: "Saved", subtitle: "5k Vacancies", logo: FaBookmark },
+  { title: "Submitted", subtitle: "12k Vacancies", logo: FaFileUpload },
+  { title: "Hired", subtitle: "8k Vacancies", logo: FaCheckDouble },
+];
+
+export default function aboutus() {
+  return (
+    <div className="bg-body">
+      <Header title="Find Your Job" subtitle="Live Your Dream " img={HeadPic} />
```
Comment (zahrayh) [inline pages/aboutus.js:52]: the header image is too small and too much empty space between text and image

Comment (rebaz36) [reply pages/aboutus.js:52]: i fixed this on @ayasalam0 issue it should be fixed once i merge hers

File: pages/aboutus.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+              post resumes, and research companies. Every day, we connect
+              millions of people to new opportunities. but of course this also
+              allows companies find the right candidate faster putting
+              themselves out there and allowing job seekers to see their open
+              positions.
+            </h2>
+          </div>
+        </div>
+
+        <h1 className="text-4xl font-semibold text-primary pt-40">
```
[the remark this answers is not in the document]
Comment (rebaz36) [reply pages/aboutus.js:81]: fixed

File: pages/aboutus.js [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+          {" "}
+          How it works
+        </h1>
+        <div className="grid grid-cols-3 p-6 gap-12">
```
[the remark this answers is not in the document]
Comment (rebaz36) [reply pages/aboutus.js:85]: done

**Comment by Ge6ben** [comment]
_Kaka why u are not using **MAP** !!! to make our life much easier!!_

```
{Sec2.map((s, id) => {
            <WhyChooseUs
              title={s[id].title}
              subtitle={s[id].subtitle}
              Logo={s[id].logo}
            />;
          })}
```

 ```
 {Sec1.map((S, id) => {
            <HowItWorks
              title={S[id].title}
              desc={S[id].desc}
              Logo={S[id].logo}
            />;
          })}
          
```
          
 u will delete **26 lines** by these steps!!!!!

**Pull request merged**